decidim-core/app/views/layouts/decidim/_mailer_logo.html.erb

```diff
@@ -1,5 +1,6 @@
 <% if organization %>
-  <%= link_to decidim.root_url(host: organization.host) do %>
+  <% url =  decidim.root_url(host: organization.host) + "?utm_source=" + organization.host + "&utm_campaign=" + newsletter_id.to_s %>
```

Comment:
Move this to a helper, maybe?

Reply:
We shouldn't couple `decidim` to google analytics. Also, some of the projects are actually using `piwik` so we should find another way to handle this. Maybe by providing a `Decidim.track_link(link, organization)` configuration in `config/initializers/decidim.rb` that allows the app implementer to choose its own way of tracking?